src/screens/ProfileScreen/elements/ProfilePicture.js

```diff
@@ -16,46 +18,57 @@ import FastImage from 'react-native-fast-image';
  *
  * @param {ProfilePicturePropsParam} props
  */
-const ProfilePicture = ({ onImageContainerClick, profilePicPath, disabledAddIcon = false}) => {
-    const __renderAddIcon = () => {
-        if(!profilePicPath || !disabledAddIcon) return <></>
 
-        return <MemoIcAddCircle width={48} height={48} style={styles.addCircle} />
-    }
-    return <View style={styles.wrapImageProfile}>
-        <TouchableNativeFeedback onPress={onImageContainerClick}>
-            <View style={styles.profileImageContainer}>
+const ProfilePicture = ({
+  onImageContainerClick,
+  profilePicPath,
+  disabledAddIcon = false,
+  karmaScore = 0
+}) => {
+  const renderAddIcon = () => {
+    if (!profilePicPath || !disabledAddIcon) return <></>;
+
+    return <MemoIcAddCircle width={48} height={48} style={styles.addCircle} />;
+  };
+  return (
+    <View style={styles.wrapImageProfile}>
+      <TouchableNativeFeedback onPress={onImageContainerClick}>
+        <CircleGradient fill={karmaScore} size={normalize(100)} width={normalize(2)}>
+          <View style={styles.profileImageContainer}>
             <Image
-                style={styles.profileImage}
-                source={{
-                uri: profilePicPath
-                    ? `${profilePicPath}`
-                    : DEFAULT_PROFILE_PIC_PATH,
-                }}
-                resizeMode={FastImage.resizeMode.stretch}
+              style={styles.profileImage}
+              source={{
+                uri: profilePicPath ? `${profilePicPath}` : DEFAULT_PROFILE_PIC_PATH
+              }}
+              resizeMode={FastImage.resizeMode.stretch}
             />
-            { __renderAddIcon() }
-            </View>
-        </TouchableNativeFeedback>
+            {renderAddIcon()}
+          </View>
+        </CircleGradient>
+      </TouchableNativeFeedback>
     </View>
-}
+  );
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `CircleGradient` component is used here to render a circular gradient background for the profile picture based on the user's karma score. This is a good use of the new component and utility function introduced in this PR.



<!-- This is an auto-generated comment by OSS CodeRabbit -->